tasks/configure-docker.yml

```diff
@@ -0,0 +1,22 @@
+# file: postgresql/tasks/configure-docker.yml
+
+- name: PostgreSQL | Update configuration - pt. 1 (pg_hba.conf)
+  template:
+    src: pg_hba.conf.j2
+    dest: "{{postgresql_conf_directory}}/pg_hba.conf"
+    mode: 0640
```

Comment:
AFAIU you needed to duplicate the `configure` tasks (include) 
because in the docker case we have to *omit* the  `owner`attribute .

Now ansible has the `omit` attribute feature, that you could try in that case:
  owner: "{{ xxx if (not docker_mode) else 'omit')}}"